packages/app/package.json

```diff
@@ -3,15 +3,17 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
+    "dev:full": "cross-env USE_LOCAL_CONTRACTS=true next dev",
```

Comment:
this command allows you to develop your front end pointing to local contracts instead of the published ones on npm